.releaserc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "conventionalcommits"
+      }
+    ],
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/changelog",
+    [
+      "@amanda-mitchell/semantic-release-npm-multiple",
+      {
+        "registries": {
+          "github": {},
+          "public": {}
+        }
+      }
+    ],
+    [
+      "@semantic-release/git",
+      {
+        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}",
+        "assets": [
+          "packages/javascript-sdk/package.json",
+          "packages/javascript-sdk/package-lock.json"
+        ]
+      }
+    ],
+    "@semantic-release/github"
+  ],
+  "branches": [
+    "+([0-9])?(.{+([0-9]),x}).x",
```

Comment:
This is for a maintenance branch - i.e. 3.3.0 is a branch name and is in "maintenance mode" so that we are no longer actively developing but maybe have bug fixes still